.github/workflows/build_and_test.yml

```diff
@@ -99,27 +99,6 @@ jobs:
       test-type: ${{ matrix.test-type }}
     secrets: inherit
 
-
-  test-mac-for-tags-everest:
-    if: github.ref_type == 'tag' # only test all variants for tags
-    strategy:
-      fail-fast: false
-      matrix:
-        test-type: [ 'test', 'integration-test', 'everest-models-test' ]
-        python-version: [ '3.8', '3.12' ]
-        os: [ 'macos-13', 'macos-14', 'macos-14-large']
```

Comment:
Have you considered aligning the Everest matrix with the ert matrix?

```
        os: [ 'macos-13', 'macos-14']
        exclude:
          - os: 'macos-13'
            python-version: '3.12'
```

I think this will resolve your issue.

Reply:
Only trying to deploy, not to fix anything..